src/BlockTypes/Checkout.php

```diff
@@ -14,6 +14,13 @@ class Checkout extends AbstractBlock {
 	 */
 	protected $block_name = 'checkout';
 
+	/**
+	 * Metadata path
+	 *
+	 * @var string;
+	 */
+	protected $metadata_path = 'cart-checkout/checkout/block.json';
```

Comment:
Do you think internally, we could restructure things a bit so it's always derived from `block-name/block.json` to avoid manually setting paths per block class? Then we can just look to see if it exists when registering the block.

Reply:
I moved checkout into its own directory and updated the function in `AssetsApi` to find metadata based on the block name, is this what you had in mind?

Reply:
Yup 👍🏻 thanks